frontend/ui/widget/buttondialog.lua

```diff
@@ -90,9 +91,15 @@ function ButtonDialog:init()
         self.width = math.floor(math.min(Screen:getWidth(), Screen:getHeight()) * self.width_factor)
     end
     if self.dismissable then
-        if Device:hasKeys() then
-            local close_keys = Device:hasFewKeys() and { "Back", "Left" } or Device.input.group.Back
-            self.key_events.Close = { { close_keys } }
+        if Device:hasKeys() then -- NB! Documentation + not adding unnecessary things to memory.
```

Comment:
ahem, that's not meant to be included ;-)

Reply:
lol, I didn't know what NB was so assumed it was something you wanted there.

Reply:
Nota bene